CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/) 
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.5.1]
+
+### Changed and Added
+* Changed the hyp3_insar_stack_for_ts_analysis.jpynb to make it work with MintPy v1.5.2
+* Added the hyp3_isce2_burst_stack_for_ts_analysis.jpynb to make the hyp3-isce2 burst insar work with MintPy
```

Comment:
These sections should be kept separate, per https://keepachangelog.com/en/1.0.0/#how
```suggestion
## [0.6.0]

### Added
* Added the hyp3_isce2_burst_stack_for_ts_analysis.jpynb to make the hyp3-isce2 burst insar work with MintPy

### Changed
* Changed the hyp3_insar_stack_for_ts_analysis.jpynb to make it work with MintPy v1.5.2
```